client/src/Plant/Plant.js

```diff
@@ -0,0 +1,23 @@
+import { Route, Switch, useRouteMatch } from "react-router-dom";
+import PlantList from './PlantList';
+import PlantForm from './PlantForm';
+
+function Plant(){
```

Comment:
Lets rename this from Plant to Plants (plural), including the filename and the containing folder.

This folder contains _multiple_ components that involve the collection of plants in the system.

This file contains the configuration for the _collection_ of endpoints (/, /new, /:id/edit) related to plants.

When we deal with multiple/collection of things, we use a plural name to help indicate that.